src/makeCalculator.js

```diff
@@ -37,7 +37,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(x) {
+      calculator.result += x;
+    },
+
+    subtract(x) {
+      calculator.result -= x;
```

Comment:
Using 'calculator' before it is fully defined might lead to undesirable results or crashes due to hoisting. Consider using 'this' to refer to the calculator object.